core/testing.go

```diff
@@ -22,7 +22,8 @@ func MockConfig(t *testing.T) *Config {
 // StartMockNode starts a mock Core node background process and returns it.
 func StartMockNode() *node.Node {
 	app := kvstore.NewApplication()
-	app.RetainBlocks = 10
+	// TODO(@Bidon15): Needs to be changed/configurable
+	app.RetainBlocks = 60
```

Comment:
Look at him, sitting there and looking like nothing has happened. How dare you!

Reply:
Yeah. True. Both went through the storm to find this 😅